conda/recipe/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# (C) Copyright IBM Corp. 2019. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Create build directory for cmake and enter it
+mkdir $SRC_DIR/build
+cd $SRC_DIR/build
+
+#Determine Architecture
+ARCH="powerpc64le-conda_cos7"
+ARCH_SHORTNAME="ppc64le"
+if [ `arch` = "x86_64" ]; then
+    ARCH="x86_64-conda_cos6"
+    ARCH_SHORTNAME="x86_64"
+fi
+
+# Create 'gcc' symlink so nvcc can find it
+ln -s $CONDA_PREFIX/bin/${ARCH}-linux-gnu-gcc $CONDA_PREFIX/bin/gcc
+
+# Add libjpeg-turbo location to front of CXXFLAGS so it is used instead of jpeg
+export CXXFLAGS="-I$CONDA_PREFIX/libjpeg-turbo/include ${CXXFLAGS}"
+
+# Build
+# BUILD_TENSORFLOW No longer exists. Previous flag to build tf plugin (used in release_v0.9)
+cmake -DBUILD_TENSORFLOW=ON \
+      -DCUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda \
+      -DCUDA_rt_LIBRARY=$CONDA_PREFIX/${ARCH}-linux-gnu/sysroot/usr/lib/librt.so \
+      -DNVJPEG_ROOT_DIR=$CONDA_PREFIX/lib64/ \
+      -DFFMPEG_ROOT_DIR=$CONDA_PREFIX/lib \
+      -DJPEG_INCLUDE_DIR=$CONDA_PREFIX/libjpeg-turbo/lib \
```

Comment:
Shouldn't this be
```suggestion
      -DJPEG_INCLUDE_DIR=$CONDA_PREFIX/libjpeg-turbo/include \
```
?

Reply:
Seems as though the `JPEG_INCLUDE_DIR` parameter may be a little misleading. Here is the error I get when adding your suggestion:
```
CMake Error at /root/anaconda3/conda-bld/dali_1565724921970/_build_env/share/cmake-3.12/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find JPEG: Found unsuitable version "", but required is at least                                        
  "62" (found
  /root/anaconda3/conda-bld/dali_1565724921970/_build_env/libjpeg-turbo/lib/libjpeg.so)                             
Call Stack (most recent call first):
  /root/anaconda3/conda-bld/dali_1565724921970/_build_env/share/cmake-3.12/Modules/FindPackageHandleStandardArgs.cmake:376 (_FPHSA_FAILURE_MESSAGE)
  /root/anaconda3/conda-bld/dali_1565724921970/_build_env/share/cmake-3.12/Modules/FindJPEG.cmake:89 (find_package_handle_standard_args)
  cmake/Dependencies.common.cmake:59 (find_package)
  cmake/Dependencies.cmake:88 (include)
  CMakeLists.txt:70 (include)
```
It seems as though the requirement for `JPEG_INCLUDE_DIR` is to be pointing to the location of the jpeg shared object file, which is located in `$CONDA_PREFIX/libjpeg-turbo/lib`

Reply:
The problem here is that the version string remains empty when you provide those JPEG_* variables explicitly. I solved the issue of finding libjpeg-turbo in a different way (see #1162) by providing a CMake prefix path (and not filling FindJPEG outputs directly):
```
cmake -DCUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda \
      -DCUDA_rt_LIBRARY=$BUILD_PREFIX/${ARCH_LONGNAME}-linux-gnu/sysroot/usr/lib/librt.so \
      -DCUDA_CUDA_LIBRARY=/usr/local/cuda/targets/${ARCH}-linux/lib/stubs/libcuda.so \
      -DNVJPEG_ROOT_DIR=$PREFIX/lib \
      -DFFMPEG_ROOT_DIR=$PREFIX/lib \
      -DCMAKE_PREFIX_PATH="$PREFIX/libjpeg-turbo;$PREFIX" \
      -DCMAKE_INSTALL_PREFIX=$PREFIX \
      ..
```